src/import-service/handlers/importFileParser.js

```diff
@@ -0,0 +1,61 @@
+const AWS = require('aws-sdk');
+const csv = require('csv-parser');
+const { S3_BUCKET } = process.env;
+
+const importFileParser = async (e) => {
+  console.log('Lambda function has been invoked with event: ', e);
+
+  try {
+    const s3 = new AWS.S3({ region: 'eu-west-1' });
+    const streams = [];
+
+    for (const record of e.Records) {
+      const { key } = record.s3.object;
+      const stream = new Promise((resolve, reject) => {
+        const s3Stream = s3.getObject({
+          Bucket: S3_BUCKET,
+          Key: key,
+        }).createReadStream();
+
+        s3Stream.pipe(csv())
+          .on('open', () => console.log(`Parsing file ${key}`))
+          .on('data', data => console.log('csv-parser data:', data))
+          .on('error', error => {
+            console.log('Error:', error);
+            reject(error);
+          })
+          .on('end', async () => {
+            console.log(`Moving from ${S3_BUCKET}/${key}`);
+            const newKey = key.replace('uploaded', 'parsed');
+            await s3.copyObject({
+              Bucket: S3_BUCKET,
+              CopySource: `${S3_BUCKET}/${key}`,
+              Key: newKey,
+            }).promise();
+            await s3.deleteObject({
+              Bucket: S3_BUCKET,
+              Key: key,
+            }).promise();
+            console.log(`Moved to ${S3_BUCKET}/${newKey}`);
+            resolve(`${S3_BUCKET}/${newKey}`);
+          });
+      });
+      streams.push(stream);
+    }
+
+    const results = await Promise.allSettled(streams);
```

Comment:
Since you are using `allSettled` method it would be better to handle rejected promises